src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -40,8 +40,18 @@ export class AddressForm extends React.Component<
   AddressFormProps,
   AddressFormState
 > {
+  // We need to pull out _only_ the values specified by the Address type,
+  // since our state will be used for Relay variables later on. The
+  // easiest way to do this is with the emptyAddress.
+  sanitizedPropsAddress: Partial<Address> = Object.keys(emptyAddress).reduce(
```

Comment:
This operation is a little confusing without a higher-level context, since `defaultValue` is already a `Partial<Address>`. Can we move it a level up (e.g. into `Shipping/index`)?

Reply:
Yeah, I waffled on that. I think moving it up makes sense 👍